test/benchmarks/driverBench/common.js

```diff
@@ -4,10 +4,72 @@ const fs = require('fs');
 const path = require('path');
 const { Readable } = require('stream');
 const { pipeline } = require('stream/promises');
-const { MongoClient } = require('../../..');
-const { GridFSBucket } = require('../../..');
-// eslint-disable-next-line no-restricted-modules
-const { MONGODB_ERROR_CODES } = require('../../../lib/error');
+const child_process = require('child_process');
+
+/**
+ * The path to the MongoDB Node.js driver.
+ * This MUST be set to the directory the driver is installed in
+ * NOT the file "lib/index.js" that is the driver's export.
+ */
+const MONGODB_DRIVER_PATH = (() => {
```

Comment:
Also happy to pull out all this revision/version stuff. I think I've adequately made it fail nicely when the files/env vars don't exist. Again, it provided value once so it could again.